server/src/utils/cosmosClient.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CosmosClient, CosmosClientOptions, Database, Container, ContainerRequest } from '@azure/cosmos';
+import { DefaultAzureCredential } from '@azure/identity';
+
+const defaultAzureCredential = new DefaultAzureCredential();
+
+const cosmosClientOptions: CosmosClientOptions = {
+  endpoint: process.env.VV_COSMOS_DB_ENDPOINT!,
+  aadCredentials: defaultAzureCredential
+};
+
+const initializeCosmosClient = () => {
+  if (process.env.VV_COSMOS_DB_CONNECTION_STRING) {
+    return new CosmosClient(process.env.VV_COSMOS_DB_CONNECTION_STRING);
+  } else {
+    return new CosmosClient(cosmosClientOptions);
+  }
+};
+
+export const cosmosClient = initializeCosmosClient();
+
+export const virtualVisitsDatabaseName = process.env.VV_COSMOS_DB_NAME!;
+export const surveyContainerName = 'Surveys';
```

Comment:
Should we have some uniqueness around `sessionId` and `userId` in database. I believe by blindly persisting what we are getting through Api request, we are opening ourselves to duplicate records, if an end user somehow submits multiple surveys for a session.

This could accordingly be checked in this class before persisting if the record exists by the sessionId and userId. If exists just return a success.

Reply:
Hmm I am not sure how to make it `sessionId` and `userId` unique to prevent the duplicate .. I will need help on this.